### DISCUSSION TITLE
Normalizer class

### DISCUSSION THREAD

**Discussion by kalashnikovisme** [Purple-Magic/tramway#59] [Ideas]
I usually wrote my code when Copilot suggested me this

```ruby
normalize :supply_time, with: TimeConverter
```

and it's a good idea. What do you think about `Normalizer` class for `TramwayForm`?

**Comment by kalashnikovisme** [comment]
@haukot @moshinaan WDYT?

**Reply by moshinaan** [reply]
Looks useful if u really can get specific symbols
